models/core/logs.sql

```diff
@@ -0,0 +1,65 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key = 'log_id',
+        tags=['core', 'logs'],
+        cluster_by=['block_timestamp']
+    )
+}}
+
+with
+
+base_txs as (
+
+  select * from {{ ref("txs_deduped") }}
+  where {{ incremental_load_filter("block_timestamp") }}
+
+),
+
+logs_raw as (
+
+    select
+
+        block_id,
+        block_timestamp,
+        tx_id as tx_hash,
+        tx:receipt:logs as full_logs
+
+    from base_txs
+
+),
+
+logs as (
+
+  select
+
+    block_id,
+    block_timestamp,
+    tx_hash,
+    value:logIndex as event_index,
```

Comment:
ideally you wanna parse the data type here too. more details here: https://docs.snowflake.com/en/user-guide/querying-semistructured.html
```suggestion
    value:logIndex::string as event_index,
```

Reply:
👍 saw the context on the Transfers PR. Thanks for the review, I'll add the type-casting to my table.  
  
@antonyip please wait to merge until the next commit.